coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -525,16 +525,22 @@ export default {
   }
 }
 
-.list-item__actions .v-btn.actions__more-actions__btn {
-  min-width: 38px !important;
-  width: 38px;
-  border-top-left-radius: 0;
-  border-bottom-left-radius: 0;
-  margin-left: 1px;
+.list-item__actions {
+  .v-btn.actions__more-actions__btn {
+    min-width: 38px !important;
+    width: 38px;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    margin-left: 1px;
```

Comment:
I understand the px for font sizes but did we want to keep spacing consistent by using rem?

Reply:
This was existing code. The diff "forgot where it was" and highlighted a lot more than actually changed.

This style is for the "split menu" (action sub-menu with delete item). Vuetify hard-codes this type of button to be 64px wide, but we want it smaller, so we override it. I've added a code comment to this effect. (Relative EM units do not play a part here.)

As for fonts, they should all be in rems... so I fixed a couple in the next commit.